src/components/tests/content-tab-assistant.test.tsx

```diff
@@ -312,22 +345,35 @@ describe( 'ContentTabAssistant', () => {
 		);
 
 		render( <ContentTabAssistant selectedSite={ runningSite } /> );
-		expect( screen.getByText( 'Welcome to our service!' ) ).toBeVisible();
-		expect( screen.getByText( 'Initial message 1' ) ).toBeVisible();
-		expect( screen.getByText( 'Initial message 2' ) ).toBeVisible();
-		expect(
-			screen.getByText( 'This conversation is over two hours old.', { exact: false } )
-		).toBeVisible();
+
+		jest.advanceTimersByTime( MIMIC_CONVERSATION_DELAY + 1000 );
```

Comment:
I think we don't need to advance timers as in this test case we haven't submitted a prompt.

Reply:
Removed.